src/cli/cmds/build.ts

```diff
@@ -23,17 +23,22 @@ import Log from '../../lib/Log';
 import * as inquirer from 'inquirer';
 import {validatePassword} from '../inputHelpers';
 
-export async function build(config: Config, log = new Log('build')): Promise<void> {
-  const jdkHelper = new JdkHelper(process, config);
-  const androidSdkTools = new AndroidSdkTools(process, config, jdkHelper);
+/**
+ * Checks if the keystore password and the key password are part of the environment prompts the
+ * user for a password otherwise.
+ *
+ * @returns {Promise<[string, string]>} A promise with a tuple where the first item is they
+ * keystore password and the second is the key password.
+ */
+async function getPasswords(): Promise<[string, string]> {
+  // Check if passwords are set as environment variables.
+  const envKeystorePass = process.env['LLAMA_PACK_KEYSTORE_PASSWORD'];
+  const envKeyPass = process.env['LLAMA_PACK_KEY_PASSWORD'];
 
-  if (!await androidSdkTools.checkBuildTools()) {
-    console.log('Installing Android Build Tools. Please, read and accept the license agreement');
-    await androidSdkTools.installBuildTools();
+  if (envKeyPass !== undefined && envKeystorePass !== undefined) {
+    return [envKeystorePass, envKeyPass];
```

Comment:
Could you add a log line here saying something like "Using passwords set in the LLAMA_PACK_KEYSTORE_PASSWORD and LLAMA_PACK_KEY_PASSWORD environmental variables."?

Reply:
done